src/Components/ResourcePicker/ResourcePicker.tsx

```diff
@@ -658,7 +658,14 @@ const ResourcePicker: React.FC<IResourcePickerProps> = ({
                     filterOption={customFilter}
                 />
                 {inputError && (
-                    <Text className={classNames.errorText} variant={'small'}>
+                    <Text
+                        className={
```

Comment:
instead of swapping class names, you can actually pass this `severe` prop into the classNames function itself so that the "Message" class uses a different color internally. That way the styles have the logic for styling instead of needing separate style classes that get swapped here.
I can point to some examples if you haven't done that before, just let me know.